eclipse-collections/src/main/java/org/eclipse/collections/impl/collector/Collectors2.java

```diff
@@ -1668,6 +1670,22 @@ private Collectors2()
                 EMPTY_CHARACTERISTICS);
     }
 
+    /**
+     * Returns a SummaryStatistics with results for int, long and double functions calculated for
+     * each element in the Stream or Collection this Collector is applied to.
+     */
+    public static <T> Collector<T, ?, SummaryStatistics<T>> summarizing(
```

Comment:
I don't see how this signature is generally applicable. What are the chances of needing exactly 1 int, 1 long and 1 double summary stats? SummaryStatistics already implements a builder pattern.